### PR TITLE
New design of Candidates' Web Page

### PR DESCRIPTION
**Pull request by JamirLeal**
### What does this PR do?

- This PR reestructures the candidates carousel component using material-ui carousel, also the grid is used to images. Open positions Cards was changed into Roles cards adding images and changing the content of the component.

#### Related issues

#79

#### Merge Checklist:

- [ ] The branch is up-to-date (i.e. rebased) with the base branch
- [ ] No code style issues

### CODE REVIEW

**Comment by AuroTB** [comment]
Looks very good!
There are some small UI/UX tweaks:
### Carousel:
- Please put some sort of black gradient over the carousel so words can be seen better (try 62%-80%).
- Make the animation merge to black when changing between the images
image
image

### Role cards
- Get rid of the black border
- Make them fit in the window
- Fix the mechanics and marketing descriptions, you can find them [here](https://docs.google.com/document/d/1Gj_1sUQd7-UKPq39VXr871Pjpt4YGsXR33XSXj0Cm-4/edit#).

### Other details
- Get rid of the "CandidatesInfo" component.
- Change the banner's content to its updated description.

PS: The carousel's height does not follow the window's height, but it actually looks pretty good; I'll talk with the design team so we can decide if it stays like that.

Again, very good job, everything looks great!

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.15",
     "@fortawesome/free-solid-svg-icons": "^5.7.2",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.9.1",
+    "@material-ui/core": "^4.11.3",
```
Comment (AuroTB) [inline package.json:11]: thanks!

File: src/components/Candidates/CandidatesCarousel/CandidatesCarousel.js [hunk 2]
```diff
@@ -1,41 +1,291 @@
...
+    fontSize: '6rem !important',
+    opacity: '1 !important',
+    color: 'white !important',
+  },
+})(NavigateBeforeIcon)
+
+function CandidatesCarousel() {
+  const classes = useStyles()
+
+  const [dimensions, setDimensions] = React.useState({
```
Comment (AuroTB) [inline src/components/Candidates/CandidatesCarousel/CandidatesCarousel.js:237]: nice!

File: src/components/Candidates/Candidates.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import positionsData from '../../data/positions.json'
 import CandidatesCarousel from './CandidatesCarousel/CandidatesCarousel'
 import CandidatesInfo from './CandidatesInfo/CandidatesInfo'
-import CandidatesJoinUs from './CandidatesJoinUs/CandidatesJoinUs'
```
Comment (AuroTB) [inline src/components/Candidates/Candidates.js:6]: You can eliminate the folder this component was in.

File: src/components/Candidates/Roles/CandidatesOpenPositions/CandidatesOpenPositions.js [hunk 4]
```diff
@@ -72,10 +73,11 @@ class CandidatesOpenPostions extends Component<Props, State> {
     const { isModalOpen, trySubmit, selectedPosition } = this.state
     return (
       <Row className="justify-content-center mt-4">
-        <Col xs="10" sm="10" md="10" lg="10" xl="10">
-          <Row>
+        <Col xs="12" sm="12" md="12" lg="12" xl="12">
```
Comment (AuroTB) [inline src/components/Candidates/Roles/CandidatesOpenPositions/CandidatesOpenPositions.js:76]: I think just keeping xs="12" is enough.

**Review by AuroTB — COMMENTED** [review]
Excelent code, looks great! Just change the UI/UX details and we're good to go!

**Review by AuroTB — CHANGES_REQUESTED** [review]
Excelent code!
Just change the UX/UI details and we are good to go!

**Comment by JamirLeal** [comment]
I made some changes related to the comments. Thanks

File: src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css [hunk 5]
```diff
@@ -1,12 +1,75 @@
...
   height: 100vh;
+  max-height: 800px;
   overflow: hidden;
 }
 
 .candidates-header-img {
   width: 100%;
   height: 100%;
   object-fit: cover;
+  opacity: 0.5;
```
Comment (AuroTB) [inline src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css:21]: change to 0.65

File: src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css [hunk 6]
```diff
@@ -1,12 +1,75 @@
+button:focus {
+  outline: none;
+}
+
+.candidates-carousel {
+  background: black !important;
```
Comment (AuroTB) [inline src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css:6]: add:
height: 100vh;

to make it aprox the same height as the window

File: src/components/Candidates/Roles/Roles.js [hunk 7]
```diff
@@ -20,7 +21,7 @@ type Props = {
   positionsData: PositionsData
 };
 
-class CandidatesJoinUs extends Component<Props> {
+class Roles extends Component<Props> {
```
Comment (AuroTB) [inline src/components/Candidates/Roles/Roles.js:24]: Please convert this to a functional component using Hooks;

Also, eliminate joinUsCallback() if not used

File: src/components/Candidates/Roles/CandidatesOpenPositions/OpenPositionCard/OpenPositionCard.css [hunk 8]
```diff
@@ -2,12 +2,12 @@
...
   flex-direction: row !important;
   color: black;
-  padding: 40px 30px;
   background-color: #13191E;
-  border-radius: 6px;
+  border-radius: 20px;
   border: none;
-  padding: 40px 30px;
-  margin: 0 3% 30px 0;
+  max-height: 280px;
```
Comment (AuroTB) [inline src/components/Candidates/Roles/CandidatesOpenPositions/OpenPositionCard/OpenPositionCard.css:8]: change to:
height: 30vh
or something like that to keep the roles section the same height as the window.

Please fix the card text as well, make it a little smaller.

**Review by AuroTB — CHANGES_REQUESTED** [review]
Please change all your classes to functional components using hooks!

File: src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css (hunk 6 above)
Comment (JamirLeal) [reply src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css:6]: Is this for the Small view or for large view?

Comment (AuroTB) [reply src/components/Candidates/CandidatesCarousel/CandidatesCarousel.css:6]: both

**Review by AuroTB — CHANGES_REQUESTED** [review]

**Review by AuroTB — APPROVED** [review]
LGTM

**Pull request merged**